src/controllers/calendar.ts

```diff
@@ -0,0 +1,195 @@
+import Reminder from '../models/calendar-reminder';
+import User from '../models/user';
+import Event from '../models/event';
+import {Request, Response, NextFunction} from 'express';
+import {createError, createResponse} from '../utils/helpers';
+
+//?Tested Ok
+// Set reminder
+export const setReminder = async (
+  req: Request,
+  res: Response,
+  next: NextFunction
+) => {
+  try {
+    // check user authentication
+    const user = await User.findById(req.payload.id);
+    if (!user) {
+      throw createError(500, 'Unauthenticated', 'Authentication Failed');
+    }
+    const reminder = new Reminder({
+      ...req.body,
+      createdBy: user._id,
+    });
+    await reminder.save();
+    res.send(createResponse('Reminder Added Succesfully', reminder));
+  } catch (err) {
+    return next(err);
+  }
+};
+
+export const getReminder = async (
```

Comment:
This seems cool for now, but ideally just sending the reminders as a JSON object won't be useful, we would need to have some kind of push notification system for this.